sunkit_pyvista/tests/conftest.py

```diff
@@ -0,0 +1,98 @@
+import os
+import inspect
+import platform
+import warnings
+from pathlib import Path
+
+import pytest
+import pyvista
+from pyvista._vtk import VTK9
+
+IMAGE_CACHE_DIR = os.path.join(Path(__file__).parent.absolute(), 'image_cache')
+if not os.path.isdir(IMAGE_CACHE_DIR):
+    os.mkdir(IMAGE_CACHE_DIR)
+# Normal image warning/error thresholds (assumes using use_vtk)
+IMAGE_REGRESSION_ERROR = 500  # major differences
+IMAGE_REGRESSION_WARNING = 400  # minor differences
+
+# Image regression warning/error thresholds for releases after 9.0.1
```

Comment:
Of VTK?

Reply:
I'm not sure I understand what you meant

Reply:
What is this comment and the variables for?

Reply:
When we discussed about copying the function from pyvista I decided to leave this in because we might have some tests falling in this range, I could remove it if we don't need it.

Reply:
If these aren't used, I don't see the need in keeping them.

Reply:
Alright, I'll make that change